Src/UIComponents/Select/Select.ts

```diff
@@ -1,68 +1,80 @@
+  const select = createElement('div', 'select-box');
+  const spanMainText = createElement('span');
+  spanMainText.appendChild(document.createTextNode(placeholder));
+  select.appendChild(spanMainText);
 
-    const optionsContainer = createElement('div', 'options');
+  const iconContainer = createElement('span');
+  iconContainer.innerHTML = ArrowDownIcon;
+  select.appendChild(iconContainer);
 
-    const setCorrectOptionStyles = (chosenKey: string) => {
-        Array.from(optionsContainer.children).forEach(optionComponent => {
-            if ((optionComponent as HTMLDivElement).dataset.key === chosenKey) {
-                optionComponent.classList.add("selected");
-            } else {
-                optionComponent.classList.remove("selected");
-            }
-        });
-    };
+  const optionsContainer = createElement('div', 'options');
 
-    options.forEach(singleOption => {
-        const {key, label} = singleOption;
-        const option = createElement('div', 'option');
-        option.appendChild(document.createTextNode(label));
-        option.setAttribute('data-key', key);
-        optionsContainer.appendChild(option);
+  const setCorrectOptionStyles = (chosenKey: string) => {
+    Array.from(optionsContainer.children).forEach((optionComponent) => {
+      if ((optionComponent as HTMLDivElement).dataset.key === chosenKey) {
+        optionComponent.classList.add('selected');
+      } else {
+        optionComponent.classList.remove('selected');
+      }
+    });
+  };
 
-        option.addEventListener("click", e => {
-            const chosenKey = (e.target as HTMLDivElement).dataset.key;
-            const chosenLabel = (e.target as HTMLDivElement).innerHTML;
-            optionsContainer.style.display = "none";
-            spanMainText.innerHTML = chosenLabel;
-            setCorrectOptionStyles(chosenKey);
-            onSelect(chosenKey);
-          });
-    })
+  options.forEach((singleOption, index) => {
+    const { key, label } = singleOption;
+    const option = createElement('div', 'option');
+    option.appendChild(document.createTextNode(label));
+    option.setAttribute('data-key', key);
+    optionsContainer.appendChild(option);
 
-    container.appendChild(select);
-    container.appendChild(optionsContainer);
+    option.addEventListener('click', (e) => {
+      const chosenKey = (e.target as HTMLDivElement).dataset.key;
+      const chosenLabel = (e.target as HTMLDivElement).innerHTML;
+      optionsContainer.style.display = 'none';
+      spanMainText.innerHTML = chosenLabel;
+      setCorrectOptionStyles(chosenKey);
+      onSelect(chosenKey);
+    });
 
-    window.addEventListener("click", () => {
-        optionsContainer.style.display = "none";
-    })
+    if (index === 0) {
+      option.click();
+    }
+  });
+  const paragraph = createElement('p');
+  const textNode = document.createTextNode('What is your weekly goal:');
+  paragraph.className='select-header';
```

Comment:
this can be achieved with createElement attributes
const paragraph = createElement('p', 'select-header', 'What is your weekly goal:');

Reply:
done